server/testutils/factories/person.ts

```diff
@@ -39,5 +39,5 @@ export const fullPersonSummaryFactory = Factory.define<FullPersonSummary>(() =>
   crn: getCrn(),
   personType: 'FullPersonSummary',
   name: faker.person.fullName(),
-  isRestricted: false,
+  isRestricted: Boolean(Math.floor(Math.random())),
```

Comment:
There's a faker utility for boolean -- `faker.datatype.boolean()`

Reply:
Yes, there is! I'm pretty sure I didn't write that code actually - it came in with a rebase and I remember resolving the conflict. Very odd!